src/conf_mode/service_pppoe-server.py

```diff
@@ -95,7 +100,10 @@ def apply(pppoe):
                 os.unlink(file)
         return None
 
-    call(f'systemctl reload-or-restart {systemd_service}')
+    if 'restart_required' in pppoe:
```

Comment:
This somehow feels like a bug in Accel-PPP if you need to restart the daemon and a reload of the config is not sufficient?

Reply:
It is not a bug, this feature hard to implement in accel-ppp, but it is present in accel-ppp feature requests. I think we have to use this workaround  till it will implemented in accel-ppp

Reply:
Thanks @DmitriyEshenko for the explanation. Can we add a comment in the code why we use this workaround? Maybe we can also add a link to Accel feature request list to easily track it?

Other then that we can merge it for sure.

Reply:
> Can we add a comment in the code why we use this workaround? Maybe we can also add a link to Accel feature request list to easily track it?

Done!